src/find.ts

```diff
@@ -0,0 +1,89 @@
+import * as fs from "fs";
+import * as vscode from "vscode";
+import * as path from "path";
+import * as os_utils from './os-utils';
+
+
+export async function findAya(context: vscode.ExtensionContext): Promise<string | null> {
+  const config = vscode.workspace.getConfiguration("aya");
+
+  if (!config.get<boolean>("lsp.enabled")) {
+    await vscode.window.showInformationMessage("Aya language server is disabled");
+    return null;
+  }
+  /*
+    let lspFatJarPath = config.get<string>("lsp.fatJar");
```

Comment:
Please feel free to remove these commented code. Git will remember them for us. There's no need to keep them for revert manually.

Reply:
Yeah. I forgot to do that.